SmartDeviceLink/SDLURLRequestTask.m

```diff
@@ -38,14 +38,13 @@ - (instancetype)initWithURLRequest:(NSURLRequest *)request completionHandler:(SD
     }
 
     _connection = [[NSURLConnection alloc] initWithRequest:request delegate:self startImmediately:NO];
-    [_connection scheduleInRunLoop:[NSRunLoop mainRunLoop] forMode:NSDefaultRunLoopMode];
-    [_connection start];
-
-    _completionHandler = completionHandler;
-
     _mutableData = [NSMutableData data];
     _response = nil;
-    _state = SDLURLRequestTaskStateRunning;
+    
+    self.completionHandler = completionHandler;
```

Comment:
This doesn't add copy semantics. I think instead it should just use `_completionHandler = [completionHandler copy];`